ops-devnet/docker-compose.yml

```diff
@@ -107,7 +107,7 @@ services:
 
   l2:
     pid: host # allow debugging
-    image: kromanetwork/geth:v0.5.0
+    image: kromanetwork/geth:mpt-d6420143
```

Comment:
Which PR will you setup Kroma MPT devnet?

Reply:
I will change it after kromanetwork/go-ethereum#121 merged

Reply:
To setup Kroma MPT devnet, we need to add flag "--KromaZKTrie=false" to l2, add `l2GenesisKromaMPTTimeOffset` to devnet-template.json, and spin up historical node also (in docker-compose.yml, __init__.py)

Reply:
I believe the issue has been resolved in https://github.com/kroma-network/go-ethereum/pull/119, so the `--KromaZKTrie=false` option is no longer necessary.

And is there a need to spin up historical node on the devnet? Could it be for testing purposes?